Fix small bug in --isArray mode

mondalspandan opened the pull request.

Ming-Yan commented:
Hi @mondalspandan thanks for adding the info back, indeed we now try to include the option where the jetveto map should obtained by jsonpog integration.  this is in #106

mondalspandan commented:
> Hi @mondalspandan thanks for adding the info back, indeed we now try to include the option where the jetveto map should obtained by jsonpog integration. this is in #106. Please let me know if things are good to merge

Hi @Ming-Yan, does this mean the relevant vetomap is automatically mapped in #106?

mondalspandan commented:
> Hi @mondalspandan , the jet veto maps root files have copied the html content. I tested locally and replaced them with
> 
> ```shell
> wget https://github.com/cms-jet/JECDatabase/raw/refs/heads/master/jet_veto_maps/Summer22_23Sep2023/Summer22_23Sep2023_RunCD_v1.root
> wget https://github.com/cms-jet/JECDatabase/raw/refs/heads/master/jet_veto_maps/Summer22EE_23Sep2023/Summer22EE_23Sep2023_RunEFG_v1.root
> ```

Thanks a lot for spotting this! I copied and did not check if they worked, unfortunately. I have updated the PR.

Ming-Yan commented:
> > Hi @mondalspandan thanks for adding the info back, indeed we now try to include the option where the jetveto map should obtained by jsonpog integration. this is in #106. Please let me know if things are good to merge
> 
> Hi @Ming-Yan, does this mean the relevant vetomap is automatically mapped in #106?

Hi @mondalspandan yes, right now it should directly take from the files in  jsonpog-integration
